index.md

```diff
@@ -82,6 +82,8 @@ title: 'Documentation'
       * [Unique](/docs/explain/other-nodes/unique)
       * [Window Aggregate](/docs/explain/other-nodes/window-aggregate)
 
+* **[Collector](/docs/collector)**
```

Comment:
Maybe it would be good if we put the settings into their own page, so the `/docs/collector` page only has the introduction notes + security notes, and then links to the configuration settings page.

In the future we should move the monitoring user instructions from the collector README into its own docs page here, and it would make sense if that we have a "Collector overview" page then.